### ISSUE TITLE
New beta breaks vuetify setup

### ISSUE THREAD

**Issue by DRoet** [vuejs/vue-cli#4821]
### Version
4.1.0-beta.0

### Environment info
```
Docker container:
Node 13.0.1
Yarn 1.91.1

```

### Steps to reproduce
1) create new project w/ 4.1.0-beta.0
2) add vuetify
3) start dev server w/ serve command

### What is expected?
Dev server starts correctly

### What is actually happening?
```sh
yarn run v1.19.1
$ vue-cli-service serve
 INFO  Starting development server...
 ERROR  Error: [VuetifyLoaderPlugin Error] No matching rule for vue-loader found.
Make sure there is at least one root-level rule that uses vue-loader.
Error: [VuetifyLoaderPlugin Error] No matching rule for vue-loader found.
Make sure there is at least one root-level rule that uses vue-loader.
    at VuetifyLoaderPlugin.apply (/opt/app/node_modules/vuetify-loader/lib/plugin.js:18:13)
    at webpack (/opt/app/node_modules/webpack/lib/webpack.js:51:13)
    at serve (/opt/app/node_modules/@vue/cli-service/lib/commands/serve.js:161:22)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
```

**Comment by DRoet** [comment]
Idm fixing this downstream in our plugin, but I just wanted to mention the breakage

**Comment by sodatea** [comment]
Thanks for the notice.
Seems it should be fixed downstream: it's broken by this line https://github.com/vuejs/vue-cli/blob/1c41371a42d1e5c2d7c1213a4297dc899b9535e6/packages/%40vue/cli-service/lib/config/base.js#L84 and this is certainly valid usage of `vue-loader`.

**Comment by DRoet** [comment]
Ah it seems this check in our loader is now broken: https://github.com/vuetifyjs/vuetify-loader/blob/master/lib/plugin.js#L14

Probably because the new `require.resolve()` way of loading `vue-loader` now returns the full path?

**Comment by DRoet** [comment]
opened up https://github.com/vuetifyjs/vuetify-loader/pull/97 with a simple solution for this, there is probably some more work that needs to be done in order for `vue-cli-plugin-vuetify` to be fully compatible w/ yarn v2 tho.

**Comment by sodatea** [comment]
Fixed in vuetify-loader 1.4.2

**Comment by swapneshkumar** [comment]
first create project using cli
Then goto  folder where your package.json exist 
and now try run command vue add vuetify
It will work without error
